walker/tar.go

```diff
@@ -64,7 +65,15 @@ func (w LayerTar) Walk(layer io.Reader, analyzeFn WalkFunc) ([]string, []string,
 				skipDirs = append(skipDirs, filePath)
 				continue
 			}
-		case tar.TypeSymlink, tar.TypeLink, tar.TypeReg:
+		case tar.TypeSymlink:
+			if filePath == "etc/os-release" {
+				osReleaseIsSymLink = true
+				continue
+			}
+			if w.shouldSkipFile(filePath) {
+				continue
+			}
```

Comment:
I don't think we need this process. We can just add `usr/lib/os-release` to `requiredFiles` in osReleaseAnalyzer.

Reply:
Done